relay-server/src/metrics_extraction/transactions/mod.rs

```diff
@@ -324,6 +335,19 @@ impl TransactionExtractor<'_> {
             .into_metric(timestamp),
         );
 
+        // Lower cardinality duration
+        metrics.project_metrics.push(
+            TransactionMetric::Duration {
```

Comment:
Should this be `TransactionMetric`?

Reply:
@jan-auer I added a new metric type DurationCoarse to match the string [here](https://github.com/getsentry/sentry/pull/56202/files#diff-c795730a1d305a53dcd95ed823c1167a473790e14dc73c160375d0e8ccf6c56fR90)